views/term_table.erb

```diff
@@ -112,12 +113,12 @@
 
                 <span class="person-card-filter">
                     <label for="filter-input" class="fa fa-search"></label>
-                    <input type="text" class="js-filter-input" id="filter-input" placeholder="Search by name, place, etc…">
```

Comment:
Removing this `id` attribute will mean that clicking on the preceding `<label>` won't focus the input any more. I assume you didn't mean to remove the `id` attribute?